lib/vue-lib/src/design-system/menus/DropdownMenuItem.vue

```diff
@@ -8,7 +8,7 @@
       clsx(
         'flex gap-xs items-center group select-none',
         noInteract ? 'text-neutral-500' : 'cursor-pointer',
-        !endLinkTo && 'children:pointer-events-none',
+        !endLinkTo && !enableSettingsEdit && 'children:pointer-events-none',
```

Comment:
New settings button `pointer-events-none` check was annoying to lock down, but we did it